my_auth/models.py

```diff
@@ -200,6 +200,7 @@ def answer_question(self, question, answer, rel):
             if first:
                 HarambeeeQuestionAnswerTime.objects.filter(harambee=self, question=question, harambee_level_rel=rel)\
                     .exclude(id=first.id).delete()
+            created = False
```

Comment:
This feels like a bandaid... your pk (4 fields in query) should stop diplicates, if it doesn't you should find the column that makes it unique and add it to the query.